graph/src/data/store/scalar.rs

```diff
@@ -9,6 +9,10 @@ use std::str::FromStr;
 
 pub use num_bigint::Sign as BigIntSign;
 
+// Caveat: The expoent is currently an i64 and may overflow.
```

Comment:
Typo: expoent -> exponent

Reply:
Ha, in portuguese we don't have that `n`.

Reply:
😁